src/Model bias analysis.ipynb

```diff
@@ -1504,477 +2504,3954 @@
+      "wiki_debias_random_cnn_v3_112 EER: 0.62\n",
+      "wiki_debias_random_cnn_v3_113 EER: 0.53\n",
+      "wiki_debias_random_cnn_v3_114 EER: 0.72\n",
+      "wiki_debias_random_cnn_v3_115 EER: 0.75\n",
+      "wiki_debias_random_cnn_v3_116 EER: 0.69\n",
+      "wiki_debias_random_cnn_v3_117 EER: 0.67\n",
+      "wiki_debias_random_cnn_v3_118 EER: 0.78\n",
+      "wiki_debias_random_cnn_v3_119 EER: 0.66\n",
+      "wiki_cnn_v3_100 EER: 0.20\n",
+      "wiki_cnn_v3_101 EER: 0.19\n",
+      "wiki_cnn_v3_102 EER: 0.56\n",
+      "wiki_cnn_v3_103 EER: 0.32\n",
+      "wiki_cnn_v3_104 EER: 0.27\n",
+      "wiki_cnn_v3_105 EER: 0.29\n",
+      "wiki_cnn_v3_106 EER: 0.44\n",
+      "wiki_cnn_v3_107 EER: 0.22\n",
+      "wiki_cnn_v3_108 EER: 0.34\n",
+      "wiki_cnn_v3_109 EER: 0.11\n",
+      "wiki_cnn_v3_110 EER: 0.40\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "wiki_cnn_v3_111 EER: 0.47\n",
+      "wiki_cnn_v3_112 EER: 0.12\n",
+      "wiki_cnn_v3_113 EER: 0.38\n",
+      "wiki_cnn_v3_114 EER: 0.29\n",
+      "wiki_cnn_v3_115 EER: 0.16\n",
+      "wiki_cnn_v3_116 EER: 0.46\n",
+      "wiki_cnn_v3_117 EER: 0.41\n",
+      "wiki_cnn_v3_118 EER: 0.08\n",
+      "wiki_cnn_v3_119 EER: 0.24\n",
+      "wiki_debias_cnn_v3_100 EER: 0.29\n",
+      "wiki_debias_cnn_v3_101 EER: 0.20\n",
+      "wiki_debias_cnn_v3_102 EER: 0.06\n",
+      "wiki_debias_cnn_v3_103 EER: 0.21\n",
+      "wiki_debias_cnn_v3_104 EER: 0.25\n",
+      "wiki_debias_cnn_v3_105 EER: 0.22\n",
+      "wiki_debias_cnn_v3_106 EER: 0.42\n",
+      "wiki_debias_cnn_v3_107 EER: 0.18\n",
+      "wiki_debias_cnn_v3_108 EER: 0.12\n",
+      "wiki_debias_cnn_v3_109 EER: 0.48\n",
+      "wiki_debias_cnn_v3_110 EER: 0.28\n",
+      "wiki_debias_cnn_v3_111 EER: 0.44\n",
+      "wiki_debias_cnn_v3_112 EER: 0.16\n",
+      "wiki_debias_cnn_v3_113 EER: 0.10\n",
+      "wiki_debias_cnn_v3_114 EER: 0.10\n",
+      "wiki_debias_cnn_v3_115 EER: 0.32\n",
+      "wiki_debias_cnn_v3_116 EER: 0.20\n",
+      "wiki_debias_cnn_v3_117 EER: 0.29\n",
+      "wiki_debias_cnn_v3_118 EER: 0.20\n",
+      "wiki_debias_cnn_v3_119 EER: 0.29\n",
+      "wiki_debias_random_cnn_v3_100 EER: 0.41\n",
+      "wiki_debias_random_cnn_v3_101 EER: 0.36\n",
+      "wiki_debias_random_cnn_v3_102 EER: 0.29\n",
+      "wiki_debias_random_cnn_v3_103 EER: 0.43\n",
+      "wiki_debias_random_cnn_v3_104 EER: 0.50\n",
+      "wiki_debias_random_cnn_v3_105 EER: 0.28\n",
+      "wiki_debias_random_cnn_v3_106 EER: 0.28\n",
+      "wiki_debias_random_cnn_v3_107 EER: 0.35\n",
+      "wiki_debias_random_cnn_v3_108 EER: 0.35\n",
+      "wiki_debias_random_cnn_v3_109 EER: 0.36\n",
+      "wiki_debias_random_cnn_v3_110 EER: 0.37\n",
+      "wiki_debias_random_cnn_v3_111 EER: 0.20\n",
+      "wiki_debias_random_cnn_v3_112 EER: 0.28\n",
+      "wiki_debias_random_cnn_v3_113 EER: 0.41\n",
+      "wiki_debias_random_cnn_v3_114 EER: 0.33\n",
+      "wiki
```

Comment:
I think so.

Reply:
Oh - this looks misplaced. The I think so is referring to your comment:

"### Weird, is this correct? the TNRs at EER look pretty comparable...
### should we be computing the EER for the overall dataset as opposed to the term subsets?"